packages/server/package.json

```diff
@@ -28,12 +30,16 @@
     "assert-ts": "^0.3.2",
     "effection": "=2.0.0-preview.5",
     "express": "^4.17.1",
+    "get-port": "^5.1.1",
     "graphql": "^15.5.0",
     "nexus": "^1.0.0",
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@effection/channel": "2.0.0-preview.4",
+    "@effection/events": "=2.0.0-preview.4",
     "@effection/mocha": "=2.0.0-preview.3",
+    "@effection/subscription": "=2.0.0-preview.4",
```

Comment:
By having `effection` in the dependency graph, we should have access to all of these  since `effection` re-exports them. (with the exception of `@effection/mocha`

Reply:
ok, definitely better to not manage these independently